v-next.sql

```diff
@@ -801,4 +801,275 @@ FROM openchpl.certified_product a
+       qms_standard_id, deleted);
+END IF;
+
+IF to_regclass('openchpl.ix_pending_cqm_certification_criteria') IS NULL THEN
+    CREATE INDEX ix_pending_cqm_certification_criteria ON openchpl.pending_cqm_certification_criteria (pending_cqm_certification_criteria_id, pending_cqm_criterion_id, 
+       certification_criterion_id, deleted);
+END IF;
+
+IF to_regclass('openchpl.ix_pending_cqm_criterion') IS NULL THEN
+    CREATE INDEX ix_pending_cqm_criterion ON openchpl.pending_cqm_criterion (pending_cqm_criterion_id, cqm_criterion_id, pending_certified_product_id, deleted);
+END IF;
+
+IF to_regclass('openchpl.ix_pending_test_participant') IS NULL THEN
+    CREATE INDEX ix_pending_test_participant ON openchpl.pending_test_participant (pending_test_participant_id, test_participant_unique_id, education_type_id, deleted, 
+       test_participant_age_id);
+END IF;
+
+IF to_regclass('openchpl.ix_pending_test_task') IS NULL THEN
+    CREATE INDEX ix_pending_test_task ON openchpl.pending_test_task (pending_test_task_id, test_task_unique_id, deleted);
+END IF;
+
+IF to_regclass('openchpl.ix_product_version') IS NULL THEN
+    CREATE INDEX ix_product_version ON openchpl.product_version (product_version_id, product_id, deleted);
+END IF;
+
+IF to_regclass('openchpl.ix_surveillance') IS NULL THEN
+    CREATE INDEX ix_surveillance ON openchpl.surveillance (id, certified_product_id, friendly_id, type_id, deleted);
+END IF;
+
+IF to_regclass('openchpl.ix_surveillance_nonconformity') IS NULL THEN
+    CREATE INDEX ix_surveillance_nonconformity ON openchpl.surveillance_nonconformity (id, surveillance_requirement_id, certification_criterion_id, nonconformity_type, nonconformity_status_id, deleted);
+END IF;
+
+IF to_regclass('openchpl.ix_surveillance_requirement') IS NULL THEN
+    CREATE INDEX ix_surveillance_requirement ON openchpl.surveillance_requirement (id, surveillance_id, type_id, certification_criterion_id, requirement, result_id, creation_date, last_modified_date, last_modified_user, deleted);
+END IF;
+
+IF to_regclass('openchpl.ix_surveillance_requirement_type') IS NULL THEN
+    CREATE INDEX ix_surveillance_requirement_type ON openchpl.surveillance_requirement_type (id, deleted);
+END IF;
+
+IF to_regclass('openchpl.ix_test_functionality') IS NULL THEN
+    CREATE INDEX ix_test_functionality ON openchpl.test_functionality (test_functionality_id, deleted);
+END IF;
+
+IF to_regclass('openchpl.ix_test_participant') IS NULL THEN
+    CREATE INDEX ix_test_participant ON openchpl.test_participant (test_participant_id, education_type_id, deleted, test_participant_age_id);
+END IF;
+
+IF to_regclass('openchpl.ix_test_procedure') IS NULL THEN
+    CREATE INDEX ix_test_procedure ON openchpl.test_procedure (test_procedure_id, deleted);
+END IF;
+
+IF to_regclass('openchpl.ix_test_standard') IS NULL THEN
+    CREATE INDEX ix_test_standard ON openchpl.test_standard (test_standard_id, deleted);
+END IF;
+
+END$$;
+
+/* DROP INDEXES SCRIPT:
```

Comment:
The reason I added the commented section to drop the indexes is in case there is a bug due to an index or a significant performance issue and we need to revert the indexes quickly--we'll have the script ready to go in this case. Given that reasoning, please let me know if I should remove the commented section and I will do it.